### ISSUE TITLE
How to remove draw root line direction GL Mapbox

### ISSUE THREAD

**Issue by achalnexus** [mapbox/mapbox-gl-directions#90]
I have draw root line direction using GL-mapbox.
then after i have remove this root line on button click.
What i to do.?
==JS Function==

mapboxgl.accessToken = 'pk.eyJ1IjoiYWNoYWxwcmFqYXBhdGkiLCJhIjoiY2lyNGkwZGsxMDFpenUybHd5bjRtMjVjeiJ9.2teTa5MmVqOW-MDpryv56w';

```
    var map = new mapboxgl.Map({
        //hash: true,
        container: 'map',
        style: 'mapbox://styles/achalprajapati/cis1byfch0008hgnitiwbym9c',
        center: [-117.1425, 32.63638889],
        zoom: 6
    });

    var directions = new mapboxgl.Directions({
        unit: 'metric',
        profile: 'driving',
        container: 'directions',
        proximity: [-117.3186111, 33.10388889]
    });


    directions.setOrigin([-117.1425, 32.63638889]);
    directions.addWaypoint(0, [-117.1425, 32.63638889]);
    directions.addWaypoint(1, [-117.195, 32.75416667]);
    directions.addWaypoint(2, [-116.5616667, 32.93583333]);
    directions.setDestination([-116.5616667, 32.93583333]);

    map.addControl(directions);
```

**Comment by mapsam** [comment]
Hi @achalnexus - a new method `removeRoutes` will become available in version `3.0.0`, which is near completion in #98

**Comment by mapsam** [comment]
@achalnexus version `3.x` is now out, which includes the [`removeRoutes` method](https://github.com/mapbox/mapbox-gl-directions/blob/28936a8f86de94a5db82e0498356cf9feb093ffb/API.md#removeroutes) for removing the direction style from your map. Please give this a try!

If I've misunderstood your question, feel free to re-open.
